prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -381,6 +381,51 @@ periodics:
           requests:
             memory: 200Mi
             cpu: 80m
+- name: kyma-gke-end-to-end-test-backup-restore
+  cron: "0 9 * * 1-5" # "At 09:00 on every day-of-week from Monday through Friday."
+  labels:
+    preset-build-master: "true"
+    preset-stability-checker-slack-notifications: "true"
+    preset-weekly-github-integration: "true"
+    <<: *gke_job_labels_template
+  decorate: true
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+    - <<: *kyma_ref
+      base_ref: master
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190129-c951cf2
+      securityContext:
+        privileged: true
+      command:
+      - "bash"
+        env:
+          - name: INPUT_CLUSTER_NAME
+            value: "integration"
+          - name: TEST_RESULT_WINDOW_TIME
+            value: "24h"
+          - name: STABILITY_SLACK_CLIENT_CHANNEL_ID
+            value: "#c4core-kyma-ci-force"
+          - name: GITHUB_TEAMS_WITH_KYMA_ADMINS_RIGHTS
+            value: "developers,cluster-access"
+          - name: KYMA_ALERTS_SLACK_API_URL
```

Comment:
I am not sure if we should setup kyma alerts - this cluster will be short-lived

Reply:
removed